spec/redirect_report_spec.rb

```diff
@@ -19,7 +25,8 @@ def self.foreach(name)
         '/etc/nginx/conf.d/test-redirects.map',
         "~*^/article/?$ /en/article;\n"
       )
-      File.write('production.log', log_line)
+      File.open('production.log', 'w') { |logfile| logfile.puts log_line }
+      # File.write('production.log', log_line)
```

Comment:
Ya, caught this in another commit, there's another smaller PR to come that'll clean that up. :)